src/components/AdminSettings.vue

```diff
@@ -71,11 +71,21 @@
 				<div>
 					<input id="demoserver" v-model="serverMode" type="radio"
 						name="serverMode" value="demo" class="radio"
-						:disabled="updating" @input="fetchDemoServers">
+						:disabled="updating || hostErrors.some(x => x)" @input="fetchDemoServers">
 					<label for="demoserver">{{ t('richdocuments', 'Use a demo server') }}</label><br>
 					<p class="option-inline">
 						<em>{{ t('richdocuments', 'You can use a demo server provided by Collabora and other service providers for giving Collabora Online a try.') }}</em>
 					</p>
+					<div v-if="hostErrors.some(x => x)" class="option-inline">
+						<em><u><p>{{ t('richdocuments', 'Your NextCloud setup is not capable of connecting to the demo servers because;') }}</p>
```

Comment:
Please just us a p tag and do the styling using CSS instead of em/u tags. As a general design comment, we only use underlining in Nextcloud for links.

Reply:
Got it. :)